python/ray/data/tests/test_size_estimation.py

```diff
@@ -220,10 +231,12 @@ def test_split_map_batches(ray_start_regular_shared):
     # Arrow block
     ctx.target_max_block_size = 20_000_000
     ds2 = ray.data.range(1000, override_num_blocks=1).map(lambda _: ARROW_LARGE_VALUE)
-    nblocks = len(ds2.map_batches(lambda x: x, batch_size=1).get_internal_block_refs())
+    bundles = ds2.map_batches(lambda x: x, batch_size=1).iter_internal_ref_bundles()
+    nblocks = sum([len(b.block_refs) for b in bundles])
     assert nblocks == 1, nblocks
     ctx.target_max_block_size = 2_000_000
-    nblocks = len(ds2.map_batches(lambda x: x, batch_size=16).get_internal_block_refs())
+    bundles = ds2.map_batches(lambda x: x, batch_size=16).iter_internal_ref_bundles()
+    nblocks = sum([len(b.block_refs) for b in bundles])
```

Comment:
Nit: Here and elsewhere
```suggestion
    nblocks = sum(len(b.block_refs) for b in bundles)
```